c/logging.c

```diff
@@ -439,47 +444,219 @@ LoggingDestination *logConfigureDestination(LoggingContext *context,
   destination->name = name;
   destination->data = data;
   destination->handler = handler;
-  destination->dumper = standardDumperFunction;
+  destination->handler2 = handler2;
+  destination->dumper = dumper != NULL ? dumper : standardDumperFunction;
   destination->state = LOG_DESTINATION_STATE_INIT;
   return destination;
 }
 
+LoggingDestination *logConfigureDestination(LoggingContext *context,
+                                            unsigned int id,
+                                            char *name,
+                                            void *data,
+                                            LogHandler handler){
+
+  return logConfigureDestination3(context, id, name, data, handler, NULL, NULL);
+}
+
 LoggingDestination *logConfigureDestination2(LoggingContext *context,
                                              unsigned int id,
                                              char *name,
                                              void *data,
                                              LogHandler handler,
                                              DataDumper dumper){
-  LoggingDestination *destination = logConfigureDestination(context, id, name, data, handler);
-  if (destination != NULL) {
-    destination->dumper = dumper;
+
+  return logConfigureDestination3(context, id, name, data, handler, dumper, NULL);
+}
+
+static char *logLevelToString(int level) {
+  switch (level) {
+    case ZOWE_LOG_SEVERE: return "CRITICAL";
+    case ZOWE_LOG_WARNING: return "WARN";
+    case ZOWE_LOG_INFO: return "INFO";
+    case ZOWE_LOG_DEBUG: return "DEBUG";
+    case ZOWE_LOG_DEBUG2: return "FINER";
+    case ZOWE_LOG_DEBUG3: return "TRACE";
+    default: return "NA";
   }
-  return destination;
 }
 
-void printStdout(LoggingContext *context, LoggingComponent *component, void *data, char *formatString, va_list argList){
+typedef struct zl_time_t {
+  char value[32];
+} zl_time_t;
+
+static zl_time_t gettime(void) {
```

Comment:
Can you name this function according to the style of this file? E.g., `getCurrentTime`.